EIPS/eip-2657.md

```diff
@@ -0,0 +1,34 @@
+---
+eip: 2657
+title: Ephemeral Testnet Yolo
+author: James Hancock (@madeoftin)
+discussion-to: https://gitter.im/ethereum/AllCoreDevs
+status: Draft
+type: Meta
+created: 2020-04-19
+---
+
+**Disclaimer: This is for testing basic infrastructure. It will be nuked. It is not for deploying dapps, nor does it define what will go into mainnet. For information on network upgrades, please follow the relevant meta EIPs and ongoing discussion on Ethereum/pm.**
+
+## Abstract
+
+The specification for Ephemeral Testnet Yolo. Clients who wish to sync need to implement the following features into their client. It is for testing basic infrastructure and will be nuked.
+
+## Specification 
+
+Name: Yolo
+ID: `YOLO-v1`
+
+  - EIP 2537
+  - EIP 2315 (current merged Specification)
+  - Option (1) of EIP-2565 
+    - ("GQUADDIVISOR is set to ‘20’ per EIP-198. We recommend changing the value of this parameter to ‘200’.”)
```

Comment:
I think this is a no-go, from what I heard from @vorot93

Reply:
Unfortunately, yes. However, we may implement option 2 of the EIP - see openethereum/openethereum#11728

Reply:
Can you clarify @vorot93 ?

Should we change it? or remove it?

Reply:
I suggest we leave this out for now.

Reply:
I followed up on this in the ACDs chat. https://gitter.im/ethereum/AllCoreDevs?at=5ec78c7727513a72fba67dd6 

With no strong objections I'll remove for now. Waiting for a bit to give people time to respond.